frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -452,6 +452,21 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     <div className="row">
       <div className="col-xs-6">
         <>
+          <div className="form-group">
+            <Tooltip content="The channel to track and receive the updates from.">
+              <h5 className="co-required">Update Channel</h5>
+            </Tooltip>
+            <RadioGroup
+              currentValue={selectedUpdateChannel}
+              items={channels.map((ch) => ({ value: ch.name, title: ch.name }))}
+              onChange={(e) => {
+                setUpdateChannel(e.currentTarget.value);
+                setInstallMode(null);
+                setTargetNamespace(null);
+                setCannotResolve(false);
```

Comment:
We shouldn't be setting these values. This introduces a bug (fixed in master by #5701).